_extras/checkout.md

```diff
@@ -126,12 +126,12 @@ community.
 
 When signing up on the [Community Discussion Etherpad][discussion], be sure to examine the description of your event carefully. Also, **take care to double check
 the time zone**, and mark your calendar after signing up! At the time of the event, the Zoom link may be found at the top
-of the Etherpad.
+of that Etherpad.
```

Comment:
```suggestion
of the discussion Etherpad.
```
Since the goal is clarity, let's go all the way to a specific term

Reply:
If the terms has to be specific, be it 'Community Discussion Etherpad' or 'Discussion Etherpad'. A new specification within the same paragraph let one wonder whether it goes about a new object and, in fact, it doesn't. I suggest no change.